jbehave-core/src/main/java/org/jbehave/core/reporters/PrintStreamOutput.java

```diff
@@ -187,6 +192,13 @@ public void comment(String step) {
         print(format("comment", "{0}\n", step));
     }
 
+    @Override
+    public void pending(PendingStep step) {
+        PendingStepMethodGenerator generator = new PendingStepMethodGenerator(keywords);
+        String pendingMethod = generator.generateMethod(step);
```

Comment:
in order to keep backward compatibility it will be better if `PendingStep` has a getter or some method that can provide generated method